packages/dap/package.json

```diff
@@ -31,5 +31,12 @@
     "@hpke/core": "^1.7.1",
     "@hpke/dhkem-x25519": "^1.5.0",
     "esbuild": "^0.24.0"
+  },
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs.js"
+    },
+    "./browser": "./dist/browser.js"
```

Comment:
I ran this through https://arethetypeswrong.github.io/, and FWIW this new subpath export doesn't resolve with `"moduleResolution": "node10"`. This moduleResolution setting [doesn't support exports at all](https://www.typescriptlang.org/docs/handbook/modules/reference.html#node10-formerly-known-as-node), so I think that's okay. Plus, if you want browser.js, you probably want `"moduleResolution": "bundler"` anyway.